.github/ISSUE_TEMPLATE/lgtm-com---false-positive.md

```diff
@@ -0,0 +1,18 @@
+---
+name: LGTM.com - false positive
+about: Tell us about an alert that shouldn't be reported
+title: LGTM.com - false positive
+labels: false-positive
+assignees: ''
+
+---
+
+**URL to the alert on the project page on LGTM.com**
```

Comment:
I think we also need to ask them to include a description as to why they think it shouldn't be included